iceoryx_binding_c/test/CMakeLists.txt

```diff
@@ -57,5 +58,5 @@ endif()
 add_executable(${PROJECT_PREFIX}_moduletests ${MODULETESTS_SRC})
 set_property(TARGET ${PROJECT_PREFIX}_moduletests PROPERTY CXX_STANDARD ${ICEORYX_CXX_STANDARD})
 target_include_directories(${PROJECT_PREFIX}_moduletests PRIVATE .)
-target_compile_options(${PROJECT_PREFIX}_moduletests PUBLIC -fPIC PRIVATE ${TEST_CXX_FLAGS})
+target_compile_options(${PROJECT_PREFIX}_moduletests PRIVATE ${TEST_CXX_FLAGS})
```

Comment:
We don't need a ``POSITION_INDEPENDENT_CODE ON` here as fPIC is only for lib builds?

Reply:
I think it's a leftover from a previous cleanup. It's not used anywhere else.